app.py

```diff
@@ -0,0 +1,19 @@
+"""Main entrypoint for the elastic beanstalk app."""
+import logging
+
+import falcon
+
+from draw_party import urls
+
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.StreamHandler())
+log.setLevel(logging.DEBUG)
+
+app = application = falcon.API()
```

Comment:
Why are you assigning both of these? I don't see `application` used anywhere.

Reply:
It was in the example, since I'm not sure what elastic beanstalk will look for and gunicorn uses either so I just assign both